companion-select-text-insert/selected-text-below.qml

```diff
@@ -17,8 +17,8 @@ Script {
     property variant settingsVariables: [
         {
             "identifier": "linkStyle",
-            "name": "Insert Link as Ref Style",
-            "description": "Insert link As Reference , Below or NoLink ?",
+            "name": "Insert Link according to the selection",
+            "description": "Below : Inserts Pages's URL under current line. Ref: Insert Page's URL as Refrence Style Linking. NoLink : without links",
```

Comment:
Don't you want to use consistent whitespaces? Like no space before `:` 😉

Reply:
fixed , i blame my non-white spaced native language.

Reply:
I can understand that. 😁